src/command_modules/azure-cli-resource/HISTORY.rst

```diff
@@ -10,6 +10,7 @@ Release History
 * `deployment delete`: Add `--no-wait` support.
 * Added `deployment wait` command.
 * Fix issue where the subscription-level `az deployment` commands erroneously appeared for profile 2017-03-09-profile.
+* `group deployment create`: Fix issue where `--parameters {}` resulted in an error.
```

Comment:
Shall we put it under a new version, since 2.1.0 has already been cut to release?

Reply:
Good point--yes!